lib/TPP/DefaultTppPasses.cpp

```diff
@@ -105,8 +101,6 @@ struct DefaultTppPasses : public DefaultTppPassesBase<DefaultTppPasses> {
 
     // Lower all Check ops.
     pm.addPass(createConvertCheckToLoopsPass());
-    // Lower all LinalgX ops.
-    pm.addPass(createLinalgXToLoopsPass());
```

Comment:
Don't we need the generalize pack pass here, too?

Reply:
I can add it. There was no packing pass in the default pipeline so no-need at the moment, but should end up there at some point.

> Some initial comments.
> 
> Overall, what's the strategy for re-enabling the XFAILed tests?

The test on perf should be fixed in #277 and then re-enables. For the bf16 we can try to move them at tensor level, what do you think @KavithaTipturMadhu ?

Reply:
Don't the gemm/conv passes add the packs themselves?

Reply:
> For the bf16 we can try to move them at tensor level, what do you think @KavithaTipturMadhu ?

Are there any BF16 packing tests left? The idea was to leave `linalgx.pack` for BF16 until @KavithaTipturMadhu had time to fix, but if we're going to remove, then we must add something in its place.